libs/agno/agno/models/base.py

```diff
@@ -226,6 +226,12 @@ def response(self, messages: List[Message]) -> ModelResponse:
                 if any(m.stop_after_tool_call for m in function_call_results):
                     break
 
+                # Log messages for the next call to the model
```

Comment:
I don't think we should do this. The idea is that the model just adds to the logs it is already producing. It works well as it is right now?